kernel-modules/BLOCKLIST

```diff
@@ -92,3 +92,4 @@
 # Since kmods are not supported anyway, block these combination
 ~.*\.el8_9\..* * mod
 ~.*\.el9_4\..* ~2\.[3-9]\..* *
+~.*\.el[89].* 2.3.0 mod
```

Comment:
Shouldn't it be merged with the one above?
```
~.*\.el8_9\..* * mod
```

Reply:
No, that one blocks kernel modules for RHEL 8.9 on any version, this one blocks kernels for RHEL 8 or RHEL 9 on 2.3.0 (which corresponds to 3.74).

Reply:
Still I think something is off. Bear with me:
* this one `.*\.el8_9\..* * mod` blocks rhel 8.9 for kernel modules of any version
* this one `~.*\.el[89].* 2.3.0 mod` blocks rhel 8.9 and rhel 8.10 for kernel modules version 2.3.0 (plus rhel 9, but it's not relevant for the point I have in mind)

But rhel 8.10 would still have back-ports from 8.9, meaning the issue we're trying to avoid using the first block will appear again. I guess the first one then has to be extended to include 8.10, is that correct?

Reply:
Probably, yes, but I would cross that bridge when we get to it.